crm_claim_type/README.rst

```diff
@@ -0,0 +1,60 @@
+CRM Claim Types
+==============
+
+This module was written to extend crm_claim module features.
+It gives a section to define a set of claim types.
+
+Installation
+============
+
+To install this module, you just need to select it from availables modules
+
+Configuration
+=============
+
+This modules do not require special configurations.
+It contains a predefined set of claim types.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to Sales > Configurations > Claims > Claim Types
```

Comment:
This is a configration, not an usage.

Usage is what you can do one time you add a configuration.